frontend/src/main/java/de/iav/frontend/controller/CreateOrEditMeetingController.java

```diff
@@ -0,0 +1,227 @@
+    private final MeetingService meetingService = MeetingService.getInstance();
+    private final TimeSlotService timeSlotService = TimeSlotService.getInstance();
+    private final SceneSwitchService sceneSwitchService = SceneSwitchService.getInstance();
+    private final AuthService authService = AuthService.getInstance();
+    private static final Logger LOG = LogManager.getLogger();
+
+    private static final String CREATE_NOT_POSSIBLE = "Erstellen nicht möglich";
+
+    private static final String UPDATE_NOT_POSSIBLE = "Aktualisieren nicht möglich";
+
+    private static final String DATE_TIME_FORMAT = "yyyy.MM.dd HH:mm 'Uhr'";
+    private static final String TIME_FORMAT = "HH:mm";
+
+    @FXML
+    public Label lLoggedInUser;
+    @FXML
+    public Button bBack;
+    @FXML
+    public Label lMeetingTitle;
+    @FXML
+    public ComboBox<String> cbLocation;
+    @FXML
+    public Button bUpdate;
+    @FXML
+    public DatePicker dpDateTime;
+    @FXML
+    public Label lDateTime;
+    @FXML
+    public TextField tfTime;
+    @FXML
+    public Label lLocation;
+
+    private User loggedUser;
+    WayToCreateOrEdit wayToCreateOrEdit;
+    ObservableList<String> locationList = FXCollections.observableArrayList();
+    private Meeting meetingToUpdate;
+    private int meetingIndex;
+
+    public void initialize() {
+        LOG.info("---->CreateMeetingController public void initialize");
+        locationList.add("Braunschweig");
+        locationList.add("Berlin");
+        locationList.add("Hannover");
+        locationList.add("Hamburg");
+        locationList.add("Magdeburg");
+        locationList.add("Wolfsburg");
+        locationList.add("Goslar");
+        cbLocation.setItems(locationList);
+    }
+
+    public void onBackButtonClick(ActionEvent event) throws IOException {
+        sceneSwitchService.switchToTimeSlotsController(event,loggedUser);
+    }
+
+    public void onCreateButtonClick(ActionEvent event) throws InterruptedException {
+        if(wayToCreateOrEdit == WayToCreateOrEdit.CREATE){
+            createOneMeeting();
+        }
+        else if(wayToCreateOrEdit == WayToCreateOrEdit.UPDATE){
+            updateOneMeeting(event);
+        }
+        else{
+            Alerts.getMessageBoxWithWarningAndOkButton(
+                    "Irgendwas ist schief gelaufen",
+                    "Weder Create noch Update Modus gesetzt. Bitte prüfen aus welche Stelle der Controller aufgerufen wird",
+                    "Weder die   Methode setUserAndEditToShow(User user) noch  setUserAndUpdateToShow(User user) " +
+                            "werden beim Aufruf des Controllers genutzt. Dies ist aber für die Funktionsweise erforderlich");
+        }
+    }
+
+    private void createOneMeeting() throws InterruptedException {
+        try {
+            if (checkDate() && checkLocation()) {
+                Meeting meetingExist = meetingService.getMeetingByDateTimeAndLocation(lDateTime.getText(), cbLocation.getValue());
```

Comment:
*meetingExists (he she it das s muss mit)

Reply:
aber s ist auch plural und hier ist es singular